app/javascript/components/widget-controller.jsx

```diff
@@ -1,16 +1,12 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import gql from 'graphql-tag';
-import { mathMod } from 'ramda';
 import { Query } from 'react-apollo';
-import styled from 'styled-components';
-import FullPanel from '@components/full-panel';
-import SidePanel from '@components/side-panel';
-import { DisconnectedMessage, LoadingMessage } from '@messages/message';
-import Twitter from '@widgets/twitter';
-import Numbers from '@widgets/numbers';
-import Weather from '@widgets/weather';
-import Traffic from '@widgets/traffic';
```

Comment:
Why would we want to remove all of the widgets from the controller?

Reply:
They moved to the WidgetDisplay - my thinking was that it would be better to group these Widget components with display functionality rather than with the query logic in the controller.